cmd/swarm/flags.go

```diff
@@ -164,10 +164,6 @@ var (
 		Name:  "topic",
 		Usage: "User-defined topic this feed is tracking, hex encoded. Limited to 64 hexadecimal characters",
 	}
-	SwarmFeedDataOnCreateFlag = cli.StringFlag{
```

Comment:
This is another suspicious one - if the flag is defined (and feeds work has been done lately), how come `unused` reports it? @jpeletier @nolash

Reply:
@holisticode git search over the codebase shows this is not used, so nothing suspicious about it IMO.